bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```
@@ -715,6 +716,14 @@ const Whiteboard = React.memo((props) => {
         return newNext;
       };
 
+      editor.store.onAfterChange = (prev, next) => {
+        if (next['selectedShapeIds'] && next['selectedShapeIds']?.some(id => id.includes('shape:BG'))) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)